tests/memoryview/memoryview.pyx

```diff
@@ -259,9 +259,9 @@ def nested_struct(NestedStruct[:] mslice):
     >>> nested_struct(NestedStructMockBuffer(None, [(1, 2, 3, 4, 5)], format="T{ii}T{2i}i"))
     1 2 3 4 5
     """
-    buf = mslice
+    cdef object cdef object buf = mslice
```

Comment:
:)

Reply:
Ooops. I did one, and then decided that I was too lazy for the rest so did a "find and replace"